### DISCUSSION TITLE
store api checkout failed

### DISCUSSION THREAD

**Discussion by goforit00** [woocommerce/woocommerce#45028] [WooCommerce, Blocks, and Site Editing]
I have a question about store api. I don't know how to solve it.
I call the checkout  like this
<img width="559" alt="image" src="https://user-images.githubusercontent.com/4726168/163170338-ebbb4b7c-c4ca-4419-a902-44d1ab38f4db.png">

but the order is failed. like this 
<img width="286" alt="image" src="https://user-images.githubusercontent.com/4726168/163170482-670c1c1b-2473-4218-958c-59adf5bb71b3.png">

but I don't know why .  Can you help me! 
thanks

**Comment by senadir** [comment]
What PayPal plugin are you using? and does that plugin support Store API? to my knowledge, only paypal standard support Checkout and Store API for now, and you need to make sure you're passing the correct data to your payment gateway and initializing it correctly on the backend.
